experiments/ext_api/word_count_demo/postgres_source

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# change pwd to this script's folder
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+if [ ! -d _env ]; then
```

Comment:
This condition is unnecessary. `make _env` won't do anything if the file/directory is in place and not listed as `.PHONY` in the `Makefile`. `make _env` is sufficient in our case, we can scrap lines 6,8,9.